pkg/cmd/cli/cmd/importimage.go

```diff
@@ -118,6 +123,10 @@ func (o *ImportImageOptions) Validate(cmd *cobra.Command) error {
 		return kcmdutil.UsageError(cmd, "you must specify the name of an image stream")
 	}
 
+	if len(o.ReferencePolicy) > 0 && o.All {
```

Comment:
How would you wanted to work? It's not a problem to support this right away. The thing is that **all** tags will get the same reference policy, including overwriting current ones.